react/features/settings/components/web/ProfileTab.tsx

```diff
@@ -198,6 +198,7 @@ class ProfileTab extends AbstractDialogTab<IProps, any> {
         return (
             <Select
                 className = { classes.bottomMargin }
+                id = 'more-language-select'
```

Comment:
You mean, in addition to the default space key? I'd argue this is not really necessary, we use default html select elements that handle keyboard interactions normally :)

Reply:
Agreed. I think this question came because I used to work on accessibility for a project that used Enter and Space keys for the same action, at least this what the client required. Thanks again for explaining!